provisioner/echo/serve.go

```diff
@@ -282,3 +287,23 @@ func Tar(responses *Responses) ([]byte, error) {
 	}
 	return buffer.Bytes(), nil
 }
+
+func filterLogResponses(config *proto.Provision_Config, response *proto.Provision_Response) (*proto.Provision_Response, bool) {
+	responseLog, ok := response.Type.(*proto.Provision_Response_Log)
+	if !ok {
+		// Pass all non-log responses
+		return response, true
+	}
+
+	if config.ProvisionerLogLevel == "" {
+		// Don't change the default behavior of "echo"
+		return response, true
+	}
+
+	provisionerLogLevel := proto.LogLevel_value[strings.ToUpper(config.ProvisionerLogLevel)]
```

Comment:
Is there a way to avoid having to upper/lowercase this? Not blocking, just curious.

Reply:
I suppose that the only way to do this is to switch to `log_level: DEBUG`, so that it's upper-case everywhere. Not sure if it's worth it.